.github/workflows/workflow.yaml

```diff
@@ -18,11 +18,11 @@ on:
       - "**.md"
 env:
   GITHUB_REPO: open-policy-agent/gatekeeper
-  IMAGE_REPO: quay.io/open-policy-agent/gatekeeper
+  IMAGE_REPO: openpolicyagent/gatekeeper
```

Comment:
nit: any reason for getting rid of the dashes? My guess is that this is OPA's pre-existing repo?

Reply:
yea same org as other projects under open-policy-agent. https://github.com/open-policy-agent/opa/blob/master/Makefile#L21

Reply:
yea, this is the main org for opa: https://hub.docker.com/u/openpolicyagent